src/sections/Home/Layer5-statement/index.js

```diff
@@ -44,7 +45,7 @@ const BannerDefault = () => {
             </div>
```

Comment:
Checkout if replacing the `svg` image on line: 21 with `StaticImage` component is working fine or not (might require some CSS tweaks)

Reply:
Update the same, in other components of the home page as well.